backend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.22.2-alpine3.19
+
+WORKDIR /src/app
+
+RUN go install github.com/cosmtrek/air@latest
```

Comment:
Why'd you install this library in docker?

Reply:
This setup helps streamline the development process by automatically rebuilding and restarting the server whenever I make code changes, which can be very useful for testing and debugging without having to restart the container manually.